orc8r/cloud/go/services/certifier/storage/storage_blobstore.go

```diff
@@ -97,6 +97,8 @@ func (c *certifierBlobstore) GetManyCertInfo(serialNumbers []string) (map[string
 }
 
 func (c *certifierBlobstore) GetAllCertInfo() (map[string]*protos.CertificateInfo, error) {
+	infos := make(map[string]*protos.CertificateInfo)
```

Comment:
Nit: use the map literal notation -- `infos := map[string]*protos.CertificateInfo{}`

Reply:
No need to change it since this function is small. But it is good to define the variable close where you are going to use it. This way it is easier to relate the var with its context